src/components/EmojiPicker/EmojiPickerButtonDropdown.tsx

```diff
@@ -75,18 +83,14 @@ function EmojiPickerButtonDropdown(props) {
     );
 }
 
-EmojiPickerButtonDropdown.propTypes = propTypes;
-EmojiPickerButtonDropdown.defaultProps = defaultProps;
 EmojiPickerButtonDropdown.displayName = 'EmojiPickerButtonDropdown';
 
-const EmojiPickerButtonDropdownWithRef = React.forwardRef((props, ref) => (
+const EmojiPickerButtonDropdownWithRef = React.forwardRef((props: EmojiPickerButtonDropdownProps, ref: React.ForwardedRef<View>) => (
     <EmojiPickerButtonDropdown
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
         forwardedRef={ref}
     />
 ));
```

Comment:
Remove this part and instead do it like in guidelines:
```ts
- [1.2](#forwardRef) **`forwardRef`**

  ```ts
  // CustomTextInput.tsx

  import { forwardRef, useRef, ReactNode, ForwardedRef } from "react";
  import { TextInput, View } from "react-native";

  export type CustomTextInputProps = {
    label: string;
    children?: ReactNode;
  };

  function CustomTextInput(props: CustomTextInputProps, ref: ForwardedRef<TextInput>) {
    return (
      <View>
        <TextInput ref={ref} />
        {props.children}
      </View>
    );
  };

  export default forwardRef(CustomTextInput);
```

(rename forwardedRef to ref)

```suggestion
```